tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- include: setup-RedHat.yml
```

Comment:
Where is this include or the support information in your meta file?

Reply:
line removed